pkg/gator/verify/runner.go

```diff
@@ -327,7 +364,32 @@ func (r *Runner) runReview(ctx context.Context, newClient func() (gator.Client,
 		Object: *toReview,
 		Source: mutationtypes.SourceTypeOriginal,
 	}
-	return c.Review(ctx, au, reviews.EnforcementPoint(util.GatorEnforcementPoint))
+
+	review, err := c.Review(ctx, au, reviews.EnforcementPoint(util.GatorEnforcementPoint))
+
+	if e != nil {
+		resultants, err := e.Expand(toReview)
+		if err != nil {
+			return nil, fmt.Errorf("expanding resource %s: %w", toReview.GetName(), err)
+		}
+
+		for _, resultant := range resultants {
+			au := target.AugmentedUnstructured{
+				Object: *resultant.Obj,
+				Source: mutationtypes.SourceTypeGenerated,
+			}
+			resultantReview, err := c.Review(ctx, au, reviews.EnforcementPoint(util.GatorEnforcementPoint))
```

Comment:
when expansion is used, does this just return one review for the first failed expanded resultant?

Reply:
I think it aggregates it with the original review?

Reply:
If there is an error, execution is interrupted. An error is defined as a runtime error, not a violation.

Violations are aggregated.